### DISCUSSION TITLE
Save version of ELN-software

### DISCUSSION THREAD

**Discussion by SteffenBrinckmann** [TheELNConsortium/TheELNFileFormat#5] [General]
What do you think of adding the version number of the ELN-software to everybody's favorite "sdPublisher" entry. That way, importing can differentiate if the structure ever changes upon an ELN update.
As far as I understood, the parent 'version' specifies that we use an ro-crate in version 1.0, which in version 2.0 might become an xyz format.

**Comment by NicolasCARPi** [comment]
`sdPublisher` takes a `Person` or `Organization` is types, and they don't have versions.

What I suggest is to use `version` for the eln version, and [`schemaVersion`](https://schema.org/schemaVersion) for the ro-crate version (even if it is already present in the `conformsTo` and `@context`, it doesn't hurt to add it there for easy grabs.

The eln crate version should be semver, but it shouldn't be the same as the software itself. Because `<ELN>` can update from v3 to v4 without changing the .eln output. So it should only be bumped when the crate format changes.
